data/src/main/java/com/teamagam/gimelgimel/data/location/repository/LocationDataRepository.java

```diff
@@ -7,18 +7,18 @@
 import com.teamagam.gimelgimel.domain.notifications.entity.ConnectivityStatus;
 import com.teamagam.gimelgimel.domain.notifications.repository.ConnectivityStatusRepository;
 import com.teamagam.gimelgimel.domain.utils.SerializedSubjectBuilder;
+import io.reactivex.subjects.Subject;
 import javax.inject.Inject;
 import javax.inject.Named;
 import javax.inject.Singleton;
-import rx.Observable;
-import rx.subjects.SerializedSubject;
+import io.reactivex.Observable;
 
 @Singleton
 public class LocationDataRepository implements LocationRepository, LocationEventFetcher {
 
   private final ConnectivityStatusRepository mGpsConnectivityStatusRepo;
   private final LocationFetcher mLocationFetcher;
-  private final SerializedSubject<LocationSample, LocationSample> mSubject;
+  private final Subject<LocationSample> mSubject;
```

Comment:
IIRC *you* already did, in your task. Should I replace it too or just count on yours...?